specs/commandline/certificate.md

```diff
@@ -0,0 +1,228 @@
+
+Here is an example of trust store directory structure:
+
+```text
+$XDG_CONFIG_HOME/notation/truststore
+    /x509
+        /ca
+            /acme-rockets
+                cert1.pem
+                cert2.pem
+                  /sub-dir       # sub directory is ignored
+                    cert-3.pem   # certs under sub directory is ignored
+
+        /signingAuthority
+            /wabbit-networks
+                cert3.crt
+```
+
+In this example, there are two certificates stored in trust store named `acme-rockets` of type `ca`. There is one certificate stored in trust store named `wabbit-networks` of type `signingAuthority`.
+
+Please be noted there will be user level trust store and system level trust store. See [Directory spec](https://github.com/notaryproject/notation/blob/main/specs/directory.md) for more details. The commands `notation certificate add` and `notation certificate delete` are performed only on user level trust store.
+
+## Outline
+
+### notation certificate
+
+```text
+Manage certificates in trust store for signature verification
+
+Usage:
+  notation certificate [command]
+
+Aliases:
+  certificate, cert
+
+Available Commands:
+  add           Add certificates to the trust store. This command only operates on User level.
+  delete        Delete certificates from the trust store. This command only operates on User level.
+  generate-test Generate a test RSA key and a corresponding self-signed certificate.
+  list          List certificates used for verification. This command operates on User level and System level.
+  show          Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains multiple certificates, then all certificates are displayed. User level has priority over System level.
+
+Flags:
+  -h, --help   help for certificate
+
+```
+
+### notation certificate add
+
+```text
+Add certificates to the trust store. This command only operates on User level.
+
+Usage:
+  notation certificate add --type <type> --store <name> [flags] <filepath...>
+
+Flags:
+  -h, --help           help for certificate
+  -s, --store string   specify named store
+  -t, --type string    specify trust store type, options: ca, signingAuthority
+
+```
+
+### notation certificate list
+
+```text
+List certificates in the trust store. This command operates on both User level and System level.
+
+Usage:
+  notation certificate list [flags]
+
+Aliases:
+  list, ls
+
+Flags:
+  -h, --help           help for list
+  -s, --store string   specify named store
+  -t, --type string    specify trust store type, options: ca, signingAuthority
+
+```
+
+### notation certificate show
+
+```text
+Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains a certificate chain, all certificates in the chain are displayed starting from the leaf. User level has priority over System level.
```

Comment:
```suggestion
Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains multiple certificates, then all certificates are displayed. User level has priority over System level.
```
Again, programmatically, this has to align with the command description shown by 'notation cert -h'.